javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistClassDeclaration.java

```diff
@@ -227,30 +218,24 @@ private String getSuperclassFQN() {
         // add the method declaration of the superclass to the candidates, if present
         getSuperClass()
                 .flatMap(ResolvedReferenceType::getTypeDeclaration)
-                .ifPresent(superclassTypeDeclaration -> {
-                    SymbolReference<ResolvedMethodDeclaration> superClassMethodRef = MethodResolutionLogic.solveMethodInType(
+                .flatMap(superclassTypeDeclaration ->
+                        MethodResolutionLogic.solveMethodInType(
                             superclassTypeDeclaration,
                             name,
                             argumentsTypes,
                             staticOnly
-                    );
-                    if (superClassMethodRef.isSolved()) {
-                        candidates.add(superClassMethodRef.getCorrespondingDeclaration());
-                    }
-                });
+                        ).getCorrespondingDeclaration()
+                ).ifPresent(candidates::add);
 
         // add the method declaration of the interfaces to the candidates, if present
         for (ResolvedReferenceType interfaceRef : getInterfaces()) {
             if (interfaceRef.getTypeDeclaration().isPresent()) {
-                SymbolReference<ResolvedMethodDeclaration> interfaceMethodRef = MethodResolutionLogic.solveMethodInType(
+                MethodResolutionLogic.solveMethodInType(
                         interfaceRef.getTypeDeclaration().get(),
                         name,
                         argumentsTypes,
                         staticOnly
-                );
-                if (interfaceMethodRef.isSolved()) {
-                    candidates.add(interfaceMethodRef.getCorrespondingDeclaration());
-                }
+                ).getCorrespondingDeclaration().ifPresent(candidates::add);
```

Comment:
minor comment - could we get these method calls to their own lines please? ditto the other `ifPresent` call above